cmd/filebeat-keeper/main.go

```diff
@@ -1,75 +1,98 @@
 package main
 
 import (
+	"crypto/sha256"
 	"flag"
 	"fmt"
+	"io"
 	"io/ioutil"
 	"os"
 	"os/exec"
 	"path/filepath"
+	"strconv"
 	"sync"
-	"syscall"
 	"text/template"
+	"time"
 
 	"gopkg.in/yaml.v2"
 
 	"github.com/caicloud/logging-admin/pkg/util/graceful"
 	"github.com/caicloud/logging-admin/pkg/util/osutil"
 
 	"github.com/caicloud/nirvana/log"
-	"gopkg.in/fsnotify/fsnotify.v1"
 )
 
 var (
-	filebeatExecutablePath = osutil.Getenv("FB_EXE_PATH", "filebeat")
-	srcConfigPath          = osutil.Getenv("SRC_CONFIG_PATH", "/config/filebeat-output.yml")
-	dstConfigPath          = osutil.Getenv("DST_CONFIG_PATH", "/etc/filebeat/filebeat.yml")
+	filebeatExecutablePath     = osutil.Getenv("FB_EXE_PATH", "filebeat")
+	srcConfigPath              = osutil.Getenv("SRC_CONFIG_PATH", "/config/filebeat-output.yml")
+	dstConfigPath              = osutil.Getenv("DST_CONFIG_PATH", "/etc/filebeat/filebeat.yml")
+	livenessProbePeriodSeconds = osutil.Getenv("LIVENESS_PROBE_PERIOD_SECONDS", "10")
```

Comment:
when you call it "liveness probe", it's easy for people to misunderstand it as K8s liveness probe; I recommend something like `HEATLTH_CHECK_INTERVAL`

Reply:
fixed